wordle-web/pages/index.vue

```diff
@@ -75,5 +75,49 @@ function onKeyup(event: KeyboardEvent) {
   } else if (event.key.match(/[A-z]/) && event.key.length === 1) {
     game.value?.addLetter(event.key.toUpperCase());
   }
+
+}
+function calcAttempts(){
+  var attempts = 0;
+  if(game.value.gameState == GameState.Won){
+    attempts = game.value.guessIndex + 1;
+  }else{
+    attempts = game.value.guesses.length + 5;
+  }
+  return attempts;
 }
+function postScore(playerNameIn: string, attemptsIn: number, timeIn: number){
+  console.log("score data " + playerNameIn + " " + attemptsIn + " " + 0);
```

Comment:
```suggestion
  console.log("score data " + playerNameIn + " " + attemptsIn + " " + timeIn);
```
This may have helped track down that 400.